net-mail/vacation/vacation-1.2.7.0-r1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="automatic mail answering program"
+HOMEPAGE="http://vacation.sourceforge.net/"
+SRC_URI="mirror://sourceforge/vacation/${P}.tar.gz"
+LICENSE="GPL-2"
+KEYWORDS="~alpha ~amd64 ~x86"
+SLOT="0"
+
+RDEPEND="virtual/mta
+	sys-libs/gdbm"
+DEPEND="${RDEPEND}
+	!mail-mta/sendmail"
```

Comment:
Soft blockers shouldn't be DEPEND-only, as package manager can't handle that smartly on its own. Either add a hard blocker here, or move this to RDEPEND (so flip DEPEND and RDEPEND around basically).